### ISSUE TITLE
needs.json not deleting 

### ISSUE THREAD

**Issue by hagopboz** [useblocks/sphinxcontrib-needs#68]
I noticed that when you already have a needs.json with all the needs, then you remove a need and re-run the needs builder it doesn't remove the need for the json and it doesn't update the number of total needs. 

If you remove the json and run needs build again it will generate the list without the removed ones, but shouldn't the list be updated with removed needs as well?

**Comment by danwos** [comment]
You are right, the removed need should disappear from the json list.

I think we need to wipe out the complete need-list for the current doc-version from the json and create it from scratch. 
So somewhere in [builder.py](https://github.com/useblocks/sphinxcontrib-needs/blob/master/sphinxcontrib/needs/builder.py#L29) we need to have something like this:
```
needs_list.load_json()
needs_list.wipe_version(config.get("version")) # NEW line
for key, need in needs.items():
   needs_list.add_need(version, need)
```

**Comment by danwos** [comment]
Fixed with new release 0.3.0
